planning/behavior_path_planner/src/scene_module/avoidance/avoidance_module.cpp

```diff
@@ -3300,12 +3309,15 @@ void AvoidanceModule::insertWaitPoint(
     return;
```

Comment:
// If we don't need to consider deceleration constraints, insert a deceleration point
  // and return immediately
    if (!use_constraints_for_decel) {
      utils::avoidance::insertDecelPoint(
        getEgoPosition(), data.to_stop_line, 0.0, shifted_path.path, stop_pose_);
      return;
    }

Reply:
Add comment line in https://github.com/autowarefoundation/autoware.universe/pull/3978/commits/0d099f4f4baaa6ff9134f77160612075a7b5c162.